js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "atk-google-address",
-  "version": "2.2.0",
+  "version": "4.0.0",
```

Comment:
should be `1.0.0` as in https://github.com/atk4/ui/blob/fadc7680ed16ce15267968524bbc3217c8ea323b/js/package.json#L3

JS is versioned using composer